.github/workflows/main.yaml

```diff
@@ -61,5 +80,23 @@ jobs:
         with:
           context: .
           tags: |
-            ${{ env.DOCKER_NAME }}:${{ env.COMMIT_SHA }}
+            ghcr.io/${{ github.repository }}:${{ github.sha }}
           push: ${{ env.PUSH_DOCKER_IMAGE }}
+
+  postman-test:
+    needs:
+      - docker
+    name: postman test
+    runs-on: ubuntu-latest
+    services:
+      test-service:
+        image: ghcr.io/${{ github.repository }}:${{ github.sha }}
```

Comment:
I'd prefer it as environment variable, but this seems not to be possible due to this bug: https://github.com/actions/runner/issues/480